AdobeStockImageAdminUi/Ui/Component/Listing/Filter/Color.php

```diff
@@ -59,7 +65,9 @@ public function __construct(
             $filterBuilder,
             $filterModifier,
             $components,
-            $data
+            $data,
+            $bookmarkManagement,
+            $request
```

Comment:
Please, inject new dependencies according to our [Backward Compatible Development Guide](https://devdocs.magento.com/contributor-guide/backward-compatible-development/)